gobblin-data-management/src/main/java/org/apache/gobblin/data/management/conversion/hive/publisher/HiveConvertPublisher.java

```diff
@@ -253,6 +273,17 @@ public void publishData(Collection<? extends WorkUnitState> states) throws IOExc
     }
   }
 
+  @VisibleForTesting
+  public static void setDestLineageInfo(WorkUnit workUnit, ConvertibleHiveDataset convertibleHiveDataset,
```

Comment:
Remove `static` so that subclass can add its own information. For test, we can initialize a `HiveConvertPublisher` object.